frontend/src/views/instructor/ddahs/style.css

```diff
@@ -39,3 +39,41 @@
     align-self: center;
     font-style: italic;
 }
+
+.instructor-ddah-import-duties-diff-list .duty-created {
+    list-style-type: "\2713";
+}
+
+.instructor-ddah-import-duties-diff-list .duty-created:after {
+    content: " (new)";
+    color: green;
+    font-weight: bold;
+}
+
+.instructor-ddah-import-duties-diff-list .duty-updated {
+    list-style-type: "\270F";
```

Comment:
Put in the actual character instead of the escape sequence. That way it's easier to understand what's going on. Also, prefer black and white characters over color emojis.